{{cookiecutter.project_slug}}/setup.py

```diff
@@ -137,9 +137,21 @@
 
 req_setup = [{%- if cookiecutter.use_pytest == 'y' %}'pytest-runner',{%- endif %}]
 
-req_test = ['coverage', 'nose', 'nose2', 'nose-htmloutput', {%- if cookiecutter.use_pytest == 'y' %} 'pytest>=3',{%- endif %} 'rednose', 'urlchecker']
+req_test = [{%- if cookiecutter.use_pytest == 'y' %} 'pytest>=3',{%- endif %}, 'pytest-cov', 'pytest-reporter-html1', 'urlchecker']
```

Comment:
Should be `[{%- if cookiecutter.use_pytest == 'y' %} 'pytest>=3', 'pytest-cov', 'pytest-reporter-html1',{%- endif %}, 'urlchecker']`.

Reply:
Covered by 51af359